libraries/abstractions/ble_hal/include/bt_hal_gatt_server.h

```diff
@@ -687,6 +698,72 @@ typedef struct
                                      uint32_t ulTransId,
                                      BTStatus_t xStatus,
                                      BTGattResponse_t * pxResponse );
+
+    /**
+     * @brief Start directed advertising to the device address passed this device should
+     * be already part of the white list
+     *
+     * On connection triggers BTConnectionCallback_t with bConnected = true.
+     *
+     * @param[in] ucServerIf Server interface, return on the callback BTRegisterServerCallback_t
+     *  after successful pxRegisterServer call.
+     *
+     * @param[in] pxBdAddr Address of the Remote device.
+     *
+     *
+     * @return Returns eBTStatusSuccess on successful call.
+     */
+    BTStatus_t ( * pxReconnect )( uint8_t ucServerIf,
+                                  const BTBdaddr_t * pxBdAddr );
+
+    /**
+     * @brief Add devices to whitelist
+     *
+     * Triggers BTWhiteListChangedCallback_t with bIsAdded = true.
+     *
+     * @param[in] ucServerIf Server interface, return on the callback BTRegisterServerCallback_t
+     *  after successful pxRegisterServer call.
+     *
+     * @param[in] pxBdAddr Address of the Remote device.
```

Comment:
Is an array of device addresses passed in ? If so can you update this doc accordingly ?

Reply:
fixed